src/style.css

```
@@ -1 +1,70 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto-500";
+  src: url(./Roboto/Roboto-Medium.ttf);
+  font-weight: 500;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  min-width: 1024px;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.logo {
+  background: url(./images/logo.png) no-repeat;
+  width: 40px;
+  height: 40px;
+  top: 50%;
+  margin: 10px 0;
+}
+
+.nav_list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav_item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav_link {
+  display: flex;
+  position: relative;
+  color: #000;
+  font-family: "Roboto-500", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: 22.65px 0;
+
+}
```

```suggestion
  padding: 23px 0;
}
```
use integers for px amount